curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/serve-an-html-file.english.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301516
 You can respond to requests with a file using the <code>res.sendFile(path)</code> method. You can put it inside the <code>app.get('/', ...)</code> route handler. Behind the scenes, this method will set the appropriate headers to instruct your browser on how to handle the file you want to send, according to its type. Then it will read and send the file. This method needs an absolute file path. We recommend you to use the Node global variable <code>__dirname</code> to calculate the path like this:
 
 ```js
-absolutePath = __dirname + relativePath/file.ext
+absolutePath = __dirname + /relativePath/file.ext
```

Comment:
```suggestion
absolutePath = path.join(__dirname, '/relativePath/file.ext')
```
Good catch, this is definitely better.  However, I think if we're going to recommend anything it should be `path.join`